js_modules/dagster-ui/packages/ui-core/src/assets/AssetWipeDialog.oss.tsx

```diff
@@ -0,0 +1,118 @@
+import {RefetchQueriesFunction} from '@apollo/client';
+// eslint-disable-next-line no-restricted-imports
+import {ProgressBar} from '@blueprintjs/core';
+import {
+  Body1,
+  Box,
+  Button,
+  Dialog,
+  DialogBody,
+  DialogFooter,
+  Group,
+  ifPlural,
+} from '@dagster-io/ui-components';
+import {memo, useMemo} from 'react';
+
+import {VirtualizedSimpleAssetKeyList} from './VirtualizedSimpleAssetKeyList';
+import {asAssetPartitionRangeInput} from './asInput';
+import {useWipeAssets} from './useWipeAssets';
+import {AssetKeyInput} from '../graphql/types';
+import {NavigationBlock} from '../runs/NavigationBlock';
+import {numberFormatter} from '../ui/formatters';
+
+export const AssetWipeDialog = memo(
+  (props: {
+    assetKeys: AssetKeyInput[];
+    isOpen: boolean;
+    onClose: () => void;
+    onComplete?: () => void;
+    requery?: RefetchQueriesFunction;
+  }) => {
+    return (
+      <Dialog
+        isOpen={props.isOpen}
+        title="Wipe materializations"
+        onClose={props.onClose}
+        style={{width: '80vw', maxWidth: '1200px', minWidth: '600px'}}
```

Comment:
One thing I forgot to do is to make `canEscapeKeyClose` and `canOutsideClickClose` equal to false when wiping is in progress.

Reply:
It looks like the way this is built, if the dialog is closed it stops wiping (vs preventing the dialog from closing). I think we should probably keep it that way until we add a dedicated cancel button for interrupting the wipe